src/components/shell/shell.js

```diff
@@ -226,6 +226,7 @@ class Shell extends Component {
     eventManager.listen(this._playerResizeWatcher, FakeEvent.Type.RESIZE, debounce(this._onWindowResize, ON_PLAYER_RECT_CHANGE_DEBOUNCE_DELAY));
     eventManager.listen(player, player.Event.FIRST_PLAY, () => this._onWindowResize());
     this._onWindowResize();
+    eventManager.listen(player, FakeEvent.Type.UI_CLICKED, () => this._startHoverTimeout());
```

Comment:
`UI_CLICKED` is fired from the shell so no need the event listener here at all. just call `_startHoverTimeout` where you fire it